dist/api_data.py

```diff
@@ -546,6 +546,29 @@ def __ge__(self, other):
             periodic checkpoints''',
             min='0', max='100000'),
         ]),
+    Config('chunk_cache', '', r'''
+        chunk cache configuration options''',
+        type='category', subconfig=[
+        Config('capacity', '10GB', r'''
+            maximum memory or storage to use for the chunk cache''',
+            min='0', max='100TB'),
+        Config('chunk_cache_evict_trigger', '90', r'''
+            chunk cache percent full that triggers eviction''',
```

Comment:
I vaguely remember suggesting this before - the word `evict` in WT means the update cache. Could you use a different word here like `eject` or `removal` ? (And throughout where we have stats, function names, etc)

Reply:
IMHO, "evict" is a very general term used for all types of caches. I hesitate using an uncommon technical term here. I'll leave the wording as is for now, perhaps with can discuss at the group meeting and see where the consensus lies.